src/NServiceBus.Core/Recoverability/Faults/ErrorQueueSettings.cs

```diff
@@ -23,10 +23,29 @@ public static string ErrorQueueAddress(this ReadOnlySettings settings)
         {
             string errorQueue;
 
-            if (settings.TryGet("errorQueue", out errorQueue))
+            if (TryGetExplicitErrorQueueAddress(settings, out errorQueue))
+                return errorQueue;
+
+            return DefaultErrorQueueName;
+        }
+
+        /// <summary>
+        /// Get the explicitly configured error queue address if one is defined.
+        /// The error queue can be configured in code using 'EndpointConfiguration.SendFailedMessagesTo()',
+        /// via the 'Error' attribute of the 'MessageForwardingInCaseOfFaultConfig' configuration section,
+        /// or using the 'HKEY_LOCAL_MACHINE\SOFTWARE\ParticularSoftware\ServiceBus\ErrorQueue' registry key.
+        /// </summary>
+        /// <param name="settings">The configuration settings of this endpoint.</param>
+        /// <param name="errorQueue">The configured error queue.</param>
+        /// <returns>True if an explict error queue have been configured.</returns>
+        /// <exception cref="Exception">When the configuration for the endpoint is invalid.</exception>
+        public static bool TryGetExplicitErrorQueueAddress(this ReadOnlySettings settings, out string errorQueue)
```

Comment:
The naming of this makes it sound like it's an "explicit error queue" vs. an error queue that has been explicitly configured. How about `TryGetExplicitlyConfiguredErrorQueueAddress`? Though that's rather long.